Source/Example Measurements/README.md

```diff
@@ -1,52 +1,32 @@
+To run the example measurement, follow these steps
+
+1. Open the LabVIEW project (.lvproj) file for the example.
+2. Open and Run the following VI to run the measurement service: `Run Service.vi`
 
 ### Stop the Measurement Service
 
-Measurement Service will be stopped automaticlly when the client frameworks calls the `ServiceManagementService\Stop Service` method on the Measurement Service. To manually trigger the stop, go to `Run Service.vi` of the respective measurements, and click on `Stop` button from the Front Panel.
+Click the Stop button on the front panel of `Run Service.vi`.
 
 ## Developer Guide
 
 ### Make Changes to the Measurement Logic
 
-Measurement Logic for the measurements are coded under `Measurement Logic.vi` under the respective libraries. The measurement logic can be modified in it. After modifying the measurement logic, run the measurement by following the `Run the Measurement Service` steps from above.
-
-### How GetMetaData and DoMeasure are triggered in the Measurement Service
-
-This is abstracted in the library, and a developer need not know how the frameworks request and recieve the response from the measurement client, and when in measurement client that Get MetaData and Measure functions are called. To understand the Measurement Service please find the details below,
-
-1. To understand the LabVIEW Service and how a service function is called on client call, please refer: https://github.com/ni/grpc-labview/blob/master/docs/BasicsTutorial.md
-
-Measurement Service and the function calls are similar to how the general LabVIEW service works as mentioned in the link above. You can find the `MeasurementService` and `ServiceManagementService` under the `<libraryName>.lvlib\Service\`. 
-
-`MeasurementService` is responsible for responding to the MetaData request and calling the `Measurement Logic.vi` when the DoMeasurement request is sent from the frameworks. These implementation can be found in: `Start Sync.vi`
-
-`ServiceManagementService` is responsible for managing the service like triggering stop event when the framework request to stop the service. These implementation can be found in: `Start Sync.vi`
+Measurement logic for the measurements is contained in `Measurement Logic.vi` measurement service library.
 
 ### How to add new UI elements
 
-User Interface is created in `Measurement UI.vi` under the `<LibraryName>.lvlib\`. The control and indicator names in the User Interface should match the `Measurement Configuration` and `Measurement Results`. If the datatype and name matches, then the data from the UI will sent to the logic before execution and the results will be published to UI after measurement is run.
+The User Interface is defined by `Measurement UI.vi` under `<LibraryName>.lvlib\`. The control and indicator names in the User Interface should match the `Measurement Configuration` and `Measurement Results`. If the datatype and name matches, then the data from the UI will
```

Comment:
Same as above. I wouldn't mention the library name. These names should be unique in the examples.